pinot-core/src/main/java/org/apache/pinot/core/query/reduce/BaseGapfillProcessor.java

```diff
@@ -226,4 +231,16 @@ protected List<Object[]> gapFillAndAggregate(List<Object[]> rows, DataSchema dat
       DataSchema resultTableSchema) {
     throw new UnsupportedOperationException("Not supported");
   }
+
+  protected String caseInsensitiveTypeString(String columnName) {
```

Comment:
(I haven't check the details of the logic, so I may be wrong):
the logic will also replace `INT_column` to `int_column` (or similar for other types) if the parameter contains such string, is this intended?